testing/iframe.js

```diff
@@ -319,25 +320,42 @@ export function createIframeWithMessageStub(win) {
    * Returns a Promise that resolves when the iframe acknowledged the reception
    * of the specified message.
    */
-  element.expectMessageFromParent = msg => {
-    return new Promise(resolve => {
-      const listener = event => {
-        let expectMsg = msg;
-        let actualMsg = event.data.receivedMessage;
-        if (typeof expectMsg !== 'string') {
-          expectMsg = JSON.stringify(expectMsg);
-          actualMsg = JSON.stringify(actualMsg);
+  element.expectMessageFromParent = (type, callback) => {
+    if (typeof type === 'function') {
```

Comment:
I like it. Note: I just found one thing, the callback function is only used for testing `expectMessageFromParent` function. Do we really need it?